packages/driver-kube-docker/src/driver/creation-driver.ts

```diff
@@ -0,0 +1,69 @@
+import { Flags, Interfaces } from '@oclif/core'
+import { MachineCreationDriver, MachineCreationDriverFactory, telemetryEmitter } from '@preevy/core'
+import { DeploymentMachine, machineFromDeployment } from './common'
+import { DriverContext, clientFromConfiguration, machineConnection, flags as machineDriverFlags } from './driver'
+
+type MachineCreationDriverContext = DriverContext & { serverSideApply: boolean }
+
+export const flags = {
```

Comment:
Maybe we shouldn't expose it as a flag for now.
This feature might not be widely used or needed, and if we allow it, we'll need to support it.

Reply:
since this feature is new with `kubectl` I feel like we should stay close to its behavior.
same logic can be applied in reverse - we can remove it if it's not used.